mbta-performance/chalicelib/historic/constants.py

```diff
@@ -36,4 +37,5 @@
     "vehicle_label",
     "event_type",
     "event_time_sec",
+    "vehicle_consist",
```

Comment:
We may not have it in the historic bundles

ValueError: Usecols do not match columns, columns expected but not found: ['vehicle_consist']

Reply:
Yep, sadly not in the historic bundles

https://mbta-massdot.opendata.arcgis.com/datasets/e2344a2297004b36b82f57772926ed1a/about